app/models/hmcts_common_platform/court_application_type.rb

```diff
@@ -0,0 +1,33 @@
+module HmctsCommonPlatform
+  class CourtApplicationType
+    attr_reader :data
+
+    def initialize(data)
+      @data = HashWithIndifferentAccess.new(data || {})
+    end
+
+    def id
+      data[:id]
+    end
+
+    def description
```

Comment:
Why not type? just avoiding _type.type?

Reply:
yeah, not to mention `type` is a super generic word that also has a meaning in programming, so it's a good one to avoid.